hangupsbot/plugins/getadmins.py

```diff
@@ -0,0 +1,22 @@
+# Original Author: kilr00y@esthar.net
+
+import hangups
+
+import plugins
+
+from hangups.user import UserList
```

Comment:
I'm not sure if we want to move away from importing directly from hangups or not. I would have said yes, but I can't think of how to convert this into a better call. I'll defer the decision to @endofline